loops/perl/code.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+my $n;
+if(@ARGV) {
+	$n = int(shift @ARGV);
+} else {
+	die "Please input an integer\n";
+}
+
+my $r = int(rand(10000));
+my @a; #let perl allocate the array
```

Comment:
To match the other implementations:

```perl
my @a = (0) x 10_000;
```

Reply:
Faster to let perl allocate it dynamically in my tests.